public/app/app.ts

```diff
@@ -225,11 +233,17 @@ export class GrafanaApp {
         await preloadPlugins(awaitedAppPlugins, pluginExtensionsRegistries, 'frontend_awaited_plugins_preload');
       }
 
-      setPluginLinksHook(createUsePluginLinks(pluginExtensionsRegistries.addedLinksRegistry));
       setPluginExtensionGetter(createPluginExtensionsGetter(pluginExtensionsRegistries));
       setPluginExtensionsHook(createUsePluginExtensions(pluginExtensionsRegistries));
-      setPluginComponentHook(createUsePluginComponent(pluginExtensionsRegistries.exposedComponentsRegistry));
-      setPluginComponentsHook(createUsePluginComponents(pluginExtensionsRegistries.addedComponentsRegistry));
+      setPluginLinksHook(usePluginLinks);
+      setPluginComponentHook(usePluginComponent);
+      setPluginComponentsHook(usePluginComponents);
+
+      this.extensionRegistries = {
+        addedLinks: pluginExtensionsRegistries.addedLinksRegistry,
+        addedComponents: pluginExtensionsRegistries.addedComponentsRegistry,
+        exposedComponents: pluginExtensionsRegistries.exposedComponentsRegistry,
+      };
```

Comment:
...and remove this assignment.

Reply:
Fixed in https://github.com/grafana/grafana/pull/92014/commits/b82f7de25753b96fc83450214810a37300c11dda.